### DISCUSSION TITLE
账户马上将要到期，支付遇困难，需要lantern团队协助。

### DISCUSSION THREAD

**Discussion by ADong0103** [getlantern/lantern-forum-cn#781] [Q&A]
我是蓝灯的老用户，现在账户还有5天就到期了。1、目前我在网页版中无法找到支付宝进行付款（我手机是IOS系统的，无法下载lantern）。你们提供的银联以及信用卡方式，我这边都显示支付失败。 2、邮件这边给你们发了好几封邮件，也没得到你们的回复。3、我期望你们能给我提供一个单独的付款连接，或者将我的账户期限延长几个月直到你们解决了支付宝的问题（给我延长的时间，可以从后续服务中扣除）       望得到回复，我邮箱46219192@qq.com

**Comment by xilijuli** [comment]
我也是这样。

**Reply by xilijuli** [reply]
345408353@qq.com 能不能帮解决一下呢？

**Comment by ADong0103** [comment]
@Mike777Khoo 我还有2天就要到期了，但是还没等来你的答复，我非常着急

**Reply by xilijuli** [reply]
不要充钱，我充过到期以后还是免费版。
